graphics/src/MeshManager.cc

```diff
@@ -1579,3 +1579,9 @@ void MeshManager::ConvertPolylinesToVerticesAndEdges(
     }
   }
 }
+MeshManager* MeshManager::Instance()
+{
+  return SingletonT<MeshManager>::Instance();
+}
+
+//////////////////////////////////////////////////
```

Comment:
place the separation line above the function definition?

Reply:
Done.